engine/classes/Elgg/Cache/SimpleCache.php

```diff
@@ -137,7 +137,7 @@ function isEnabled() {
 	 */
 	function enable() {
 		$this->datalist->set('simplecache_enabled', 1);
-		$this->config->set('simplecache_enabled', 1);
+		$this->config->save('simplecache_enabled', 1);
```

Comment:
The storage is done in the datalists table, which is read into config on the next request. So this isn't necessary.